README.md

```diff
@@ -203,7 +202,7 @@ The [project factory](https://github.com/terraform-google-modules/terraform-goog
 #### Kubectl
 - [kubectl](https://github.com/kubernetes/kubernetes/releases) 1.9.x
 #### Terraform and Plugins
-- [Terraform](https://www.terraform.io/downloads.html) 0.11.x
+- [Terraform](https://www.terraform.io/downloads.html) 0.12
```

Comment:
We should add a note like we did in https://github.com/terraform-google-modules/terraform-google-network/pull/55 pointing people to the right place for a 0.11.x-compatible version.

Reply:
Added in 6cc0bf8.